### PR TITLE
Add a note in 8.8 docs about known legacy action migration issue

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/3572.

[Preview](https://security-docs_3722.docs-preview.app.elstc.co/guide/en/security/master/release-notes-header-8.8.0.html) - Added the same known issue summary to all 8.8.x releases and the 8.9.1 release notes.

### CODE REVIEW

**Review by e40pud — APPROVED** [review]
Looks great! Thanks for adding docs for this issue!

**Review by jmikell821 — APPROVED** [review]
LGTM!

**Comment by e40pud** [comment]
@nastasha-solomon I just realised that the documentation we are trying to add here are for the fix that we added in 8.9.1+, in 8.8 we have a data loss issue which we should report as a known issue.

In 8.8 `Rule changes can’t be saved if the rule’s action frequency is shorter than the rule’s run interval.`, but also such legacy actions will be removed (causing data loss).

File: docs/release-notes/8.8.asciidoc [hunk 1]
```diff
@@ -9,6 +9,7 @@
 [[known-issue-8.8.2]]
 ==== Known issues
 
+* Rule changes can't be saved if the rule's action frequency is shorter than the rule's run interval.
```
Comment (nastasha-solomon) [inline docs/release-notes/8.8.asciidoc:12]: @e40pud thanks for double-checking this. How does look?

```suggestion
* Rule changes can't be saved and existing rule actions are removed if the action's frequency is shorter than the rule's run interval.
```

Comment (e40pud) [reply docs/release-notes/8.8.asciidoc:12]: Perfect! Thank you!

**Pull request merged**